README.md

```diff
@@ -1,4 +1,4 @@
-# Flight Tracking ERAU SE300 ![shield](https://img.shields.io/badge/crates.io-v0.10.8-orange) ![shield](https://img.shields.io/badge/dependencies-secure-success) ![shield](https://img.shields.io/badge/build-success-success) ![shield](https://img.shields.io/badge/doc-passing-success)
+# Flight Tracking ERAU SE300 ![shield](https://img.shields.io/badge/crates.io-v0.10.8-orange) ![shield](https://img.shields.io/badge/dependencies-secure-success) ![shield](https://img.shields.io/badge/build-success-success) ![shield](https://img.shields.io/badge/doc-passing-success) ![shield](https://img.shields.io/badge/license-GPL-brightgreen)

```

Comment:
wtf is this. The whole point of lables is to update if things break and to link to crates.io or our actions pipeline. These are just hardcoded to always be green. You have to change the URL so that it can load the state of our project for example this link gives you the status of develop: `https://shields.io/github/workflow/status/FlightTrackingERAU/FlightTracking/CI/develop`. They have a whole list of the different ones you can do here: `https://shields.io/category/build`